### ISSUE TITLE
Some 'additional features' not displaying in both main map and report map

### ISSUE THREAD

**Issue by marsmith** [USGS-WiM/StreamStats#832]
'mainflowpath' not displaying below

![image](https://user-images.githubusercontent.com/6529123/30379961-2d4f53ac-9866-11e7-980e-3947f66176af.png)

![image](https://user-images.githubusercontent.com/6529123/30379943-19fe82aa-9866-11e7-8bda-3e4fe57e0ce9.png)

**Comment by marsmith** [comment]
Looks like some issue with the 'line' feature type.... I can't see anything obvious.  I'm going to assign to papa J since he implemented this

**Comment by marsmith** [comment]
Ok guys, i think I isolated this: the features for the lines seem to be in UTM coordinates (in NY test point).  The 10/85 point features are in geographic coordinates.  So I think we have a server side issue here.  I checked over the code and it seems like its fine--- assuming we have a good set of line coordinates.

![image](https://user-images.githubusercontent.com/6529123/35873952-36a89a1c-0b39-11e8-9246-e65472bdb2e3.png)
